linux/ibus-keyman/src/keymanutil.c

```diff
@@ -416,47 +445,37 @@ keyman_put_options_todconf(gchar *package_id,
 
     // Obtain keyboard options from DConf
     gchar **options = keyman_get_options_fromdconf(package_id, keyboard_id);
```

Comment:
Do you want to match l. 397?
```suggestion
    g_auto(GStrv) options = keyman_get_options_fromdconf(package_id, keyboard_id);
```